crates/arrow_sql_gen/src/postgres.rs

```diff
@@ -219,7 +235,7 @@ fn map_column_type_to_data_type(column_type: &Type) -> DataType {
         Type::TEXT => DataType::Utf8,
         Type::BOOL => DataType::Boolean,
         // TODO: Figure out how to handle decimal scale and precision, it isn't specified as a type in postgres types
-        Type::NUMERIC => DataType::Decimal128(38, 10),
+        Type::NUMERIC => DataType::Decimal128(38, 9),
```

Comment:
Surely there is a way to get this? It's part of the Numeric type.

Reply:
Unfortunately it's not exposed in postgres `Row` :( The only solution I've found thus far is to query the information schema ourselves, but I'm hoping to find a better solution.

Reply:
We can at least get the `scale` be reusing some of the `FromSql` logic below.